### ISSUE TITLE
[COGNITION] Global Timeout Modal - Language should not depend on color descriptions like "blue button" for users to extend their session

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16301]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Adjust language in the timeout modal.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

Our timeout modal uses language that references a "blue button" to extend user's session. This assumes the user can understand and visualize the color blue. As a user who might have a color acuity issue, I would like this to be more plain language.

## Details

VA 508 office flagged this issue on the VAOS Express Care update. I think we could change the language to something like 

> If you need more time, please select the "I need more time" button below. Otherwise we'll sign you out to protect your privacy.

Screenshot and code snippet below.

## Acceptance Criteria

- [x] Content specialist approves the text change
- [x] Change is communicated broadly in Slack
- [x] Change is verified in Staging

## Environment
* Staging env ( staging.va.gov )

## Steps to Recreate

1. Log into staging.va.gov with any known good test user
1. Leave the window / session alone for 20-30 minutes
1. The session ending modal will appear

## Solution (if known)
* https://github.com/department-of-veterans-affairs/vets-website/blob/1c1ebe64db683474fbd530f34d8853c0a20aae60/src/platform/user/authentication/components/SessionTimeoutModal.jsx#L92
```diff
! SessionTimeoutModal.jsx#L92

<p>
- If you need more time, please click on the blue button below. Otherwise, we’ll sign you out to protect your privacy.
+ If you need more time, please select the "I need more time" button below. Otherwise we'll sign you out to protect your privacy.
</p>
```

## WCAG or Vendor Guidance (optional)

* [Sensory Characteristics: Understanding SC 1.3.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-understanding.html)

## Screenshots or Trace Logs

![Screen Shot 2020-11-17 at 3 10 19 PM](https://user-images.githubusercontent.com/934879/99454667-5c3fa580-28ec-11eb-87ea-9f1d6bfdac0d.png)

**Comment by peggygannon** [comment]
Thanks, @1Copenut I will recommend updated content

Where does this alert appear? Is it a global alert across multiple applications?

**Comment by 1Copenut** [comment]
Yes, this appears anywhere users are logged in, and their session is about to expire.

**Comment by peggygannon** [comment]
hi, @1Copenut What are your thoughts about the alert having 1 button and 1 link? It seems like both actions are of equal importance to the user, so wouldn't 2 buttons be more appropriate for this alert? Let me know your thoughts! 

Recommended alert language: 

If you need more time, click I need more time below. Otherwise, we'll sign you out of your account to protect your privacy. 

{CTA buttons}
I need more time

Sign out

**Comment by 1Copenut** [comment]
@peggygannon that's a good point. I did a quick analysis and the "Logout" is a link on our global header. It's using JavaScript to trigger a redirect without a proper `href`, but it is a link. The more time and logout do seem like equal weighted actions, but my hunch is design wants to encourage extending time as the primary action, thus the larger visual prominence. 

@rtwell any thoughts or guidance here?

**Comment by AGarcia-Clarity** [comment]
I have created a [Draft PR](https://github.com/department-of-veterans-affairs/vets-website/pull/15140) that updates the SessionTimeoutModal.jsx content to what was specified above.

@1Copenut I will wait until a determination is made on which route we will take (2 buttons) vs (1 button, 1 link).

@peggygannon I have updated the language to what you provided above.

**Comment by peggygannon** [comment]
Thanks, @AGarcia-Clarity the updated message looks good. I think we should probably leave the 1 button, 1 link until we hear otherwise from design. @1Copenut brings up a good point that the desired behavior is likely to extend their session time.

**Comment by 1Copenut** [comment]
Thank you @peggygannon !

**Comment by AGarcia-Clarity** [comment]
@1Copenut so am I understanding that we're just updating the content right now (and closing this out) until we get button vs link debacle?

**Comment by 1Copenut** [comment]
@AGarcia-Clarity Yes, as far as I can tell. The original issue was the use of "blue" to identify the button, so adjusting content gets us aligned with the VA 508 office.
